rules/no-duplicate-literal-elements.js

```diff
@@ -0,0 +1,116 @@
+'use strict';
+const {newExpressionSelector} = require('./selectors/index.js');
+
+const MESSAGE_ID = 'no-duplicate-literal-elements';
+const messages = {
+	[MESSAGE_ID]: 'Remove duplicate {{valueType}} `{{value}}` from the `{{objectType}}`.',
+};
+
+const arraySelector = [
+	'VariableDeclarator > ArrayExpression > Literal',
+	'ExpressionStatement > ArrayExpression > Literal',
+	'AssignmentExpression > ArrayExpression > Literal',
+].join(',');
```

Comment:
I'm not sure about this, maybe we should check all arrays? Not just declaration and assignments?

@sindresorhus @futpib

Reply:
If I check all arrays, do I need to rule out certain situations?

For example: 
`Object.entries()` method returns an array of a given object's own enumerable string-keyed property [key, value] pairs

Reply:
I understand that, I'm not sure.

Reply:
> I'm not sure about this, maybe we should check all arrays

👍